README.md

```diff
@@ -4,50 +4,18 @@
 Central storage, management and access for important geospatial datasets
 Developed by [Land Information New Zealand](https://github.com/linz)
 
+## Development setup
 
-## Dependencies Installation
-### Python Virtual Environment (for Python CLI and AWS CDK)
-* Create and activate a Python virtual environment
+One-time setup:
 
-    ```bash
-    $ python3 -m venv .venv
-    $ source .venv/bin/activate
-    ```
-* Upgrade pip
-
-    ```bash
-    $ pip install --upgrade pip
-    ```
-* [Install Poetry](https://python-poetry.org/docs/#installation)
-* Install the dependencies:
-
-    ```bash
-    $ poetry install --extras='cdk datasets-endpoint'
-    ```
```

Comment:
I would prefer copy/paste instructions for Ubuntu rather then those URL to installation instructions which are not always very clear. For example in case of Pyenv installation it was not very helpful to me. I rather went to `https://github.com/pyenv/pyenv-installer` and did `$ curl https://pyenv.run | bash` and then `$ pyenv install`. So for me having those commands in our instructions would be much better.

Reply:
The problem with copying the instructions is that they might change upstream at any point. Would it be better to suggest an improvement to the documentation upstream maybe?